### PR TITLE
Added warning for 'vagrant destroy' command (Fixes #2003)

### LINKED ISSUE #2003: Add a warning for destroy command
### Problem

Readers should be warned of `vagrant destroy` command. This is added under vagrant commands but it is appropriate to be added as a note or warning so that to be wary before executing this command.

### Steps to reproduce the problem

N/A

### Screenshots

N/A

### Proposed solution

An additional note or warning is required in the vagrant command section about `vagrant destroy`.

### PR DESCRIPTION
**Pull request by VishalSundarrajan**
Fixes #2003 

### Description
Added a note to highlight the `vagrant destroy` command. 

### RawGit preview link

[rawgit](https://rawgit.com/)

### CODE REVIEW

**Comment by svlesiv** [comment]
Hello @VishalSundarrajan, 
Thank you for creating the issue.
However, seems that the danger of 'vagrant destroy' command has been sufficiently explained.



Could you create some other issue to work on?

**Comment by VishalSundarrajan** [comment]
@svlesiv Sure. I did because so many of them posted in git chat to rebuild BeLL after having accidentally executing this command. Hence, thought better be a note.

**Comment by svlesiv** [comment]
Hi @VishalSundarrajan, you are right.
But could you please create a note after original  text?



NOTE: Since you have already configured BeLL, you might not want to use `vagrant destroy` command. This will delete the virtual machine entirely and you have to re-configure community BeLL.

**Comment by VishalSundarrajan** [comment]
@svlesiv . In fact thats what I did

**Comment by svlesiv** [comment]
Seems that you combined everything in one note:

NOTE: Since you have already configured BeLL, you might not want to use `vagrant destroy` command. This will delete the virtual machine entirely and you have to re-configure community BeLL. To conserve hardware resources of your computer, you could use `vagrant halt` to shutdown the virtual machine that's running your community BeLL. Use `vagrant up` to turn the virtual machine back on when you need to access your community BeLL again.

Maybe, I don't see some other files.

**Comment by VishalSundarrajan** [comment]
Hi @svlesiv . I am just missing something. Can you please let me know how exactly should it look like?

File: pages/vi/vi-vagrant.md [hunk 1]
```diff
@@ -62,7 +62,9 @@ The above output tells us:
 - [`vagrant halt`](https://www.vagrantup.com/docs/cli/halt.html) will shut down the "running" virtual machine, its state will become "poweroff".
 - [`vagrant destroy`](https://www.vagrantup.com/docs/cli/destroy.html) will delete the virtual machine entirely. To use community BeLL again, we will have to rebuild a new machine from scratch using `vagrant up`.
 
-To conserve hardware resources of your computer, you could use `vagrant halt` to shutdown the virtual machine that's running your community BeLL. Use `vagrant up` to turn the virtual machine back on when you need to access your community BeLL again.
+NOTE: Since you have already configured BeLL, you might not want to use `vagrant destroy` command. This will delete the virtual machine entirely and you have to re-configure community BeLL. To conserve hardware resources of your computer, you could use `vagrant halt` to shutdown the virtual machine that's running your community BeLL. Use `vagrant up` to turn the virtual machine back on when you need to access your community BeLL again.
```
Comment (lmmrssa) [inline pages/vi/vi-vagrant.md]: @VishalSundarrajan **NOTE** you added should be for vagrant destroy. This changes is also putting `vagrant halt` command under **NOTE** so could you just more NOTE to its appropriate place without touching next line.

**Comment by svlesiv** [comment]
Hi @VishalSundarrajan, please take a look on my second comment. It illustrates how it should look like and the content of your note.

**Comment by lmmrssa** [comment]
Been there for long and not updated

**Pull request closed without merging**